.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    ignore:
+      - dependency-name: "govuk-frontend"
+    commit-message:
+      prefix: "fix"
+      prefix_development: "chore"
+      include: "scope"
```

Comment:
This will give `devDependencies` a commit with `chore(list of deps)` and production `dependencies` a commit with `fix(list of deps)`

Is this what we would like?

Reply:
It feels like they are both a `chore`, unless of course increasing the version _is_ fixing a bug?

Reply:
Are they not.... `build`? 

I don't like adding the dependencies to the change log. Except for the govuk and moj frontend packages (and someone disagreed with me on that).

Oh, we're ignoring them. Ignore this comment then. Except for the build bit ;)

Reply:
Yeah, `build` also makes sense.

Reply:
This is now configured to be in the format of `build(express)`